Added Russian translation for ISOM symbol set

sikmir opened the pull request.

dg0yt commented:
Nice.

Some symbols lost there description: 999 OpenOrienteering Logo, 799 Simple Orienteering Course, 709.2 Out-of-bounds area, dashed boundary, 709.1 Out-of-bounds area, solid boundary. (I didn't check them all.)

sikmir commented:
I've added description for those ones (999, 799, 709.2, 709.1) and rebased. Some symbols lack there description in the original ISOM_15000.xmap as well (301.2, 309.2, 310.1, 311.1, 526.1, 529.1, 529,2, 601.1, 601.3).

dg0yt commented:
Alright. Judging from the code numbers, these other symbols are subsymbols.
